### PR TITLE
Add calibrated skin

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
⚠️ This PR needs to be merged before https://github.com/robotology/walking-teleoperation/pull/113

Thanks to this PR it will be possible to use the calibrated skin provided by the skin manager to detect the roughness of an object.
The network model was provided at https://github.com/robotology/walking-teleoperation/pull/114/commits/c33ecc7f6f1cbae2d769dd0f35560b09c72ca42f. The code used to generate the model can be found in https://github.com/paolo-viceconte/rock_classifier/commit/381fcf4d68ddbdf0096b3624fb9c469174285536 (@paolo-viceconte)

Moreover this PR remove the fingertip actuation (@RiccardoGrieco) in https://github.com/robotology/walking-teleoperation/pull/114/commits/63aff826e30aa1c114e289f91a581013d72c0db5

Last but not least fixes the CI

### CODE REVIEW

File: app/robots/iCubGenova09/leftFingersHapticRetargetingParams.ini [hunk 1]
```diff
@@ -149,7 +152,7 @@ axesJointsCoupled     1
 
 # in case a calibration phase necessary before starting teleoperating the robot,
 # otherwise read the calibration matrix from the config file
-doCalibration           1
+doCalibration           0
```
Comment (S-Dafarra) [inline app/robots/iCubGenova09/leftFingersHapticRetargetingParams.ini:155]: Do you remember why this change is needed? (@RiccardoGrieco @lrapetti)

File: app/robots/iCubGenova09/rightFingersHapticRetargetingParams.ini [hunk 2]
```diff
@@ -145,7 +145,7 @@ analog_sensors_raw_max_boundary ( 25.0  0.0   37.0    15.0  24.0  12.0     0.0
 # in case each joint does not have independant motor to actuate, they are coupled
 axesJointsCoupled     1
 
-doCalibration           1
+doCalibration           0
```
Comment (S-Dafarra) [inline app/robots/iCubGenova09/rightFingersHapticRetargetingParams.ini:148]: Same as above

File: modules/HapticGlove_module/src/GloveWearable.cpp [hunk 3]
```diff
@@ -334,6 +334,7 @@ bool GloveWearableImpl::getFingertipPoseValues(Eigen::MatrixXd& values)
 
 bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>& values)
 {
+    return true;
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/GloveWearable.cpp:337]: As discussed F2F, it would be nice to have a parameter to turn this on and off, rather than exiting prematurely. @RiccardoGrieco @lrapetti do you remember why this was needed?

File: modules/HapticGlove_module/src/GloveWearable.cpp [hunk 4]
```diff
@@ -362,6 +363,7 @@ bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>&
 
 bool GloveWearableImpl::setFingertipVibrotactileValues(const std::vector<int>& values)
 {
+    return true;
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/GloveWearable.cpp:366]: As above

File: modules/HapticGlove_module/src/RobotController.cpp [hunk 5]
```diff
@@ -62,10 +62,10 @@ bool RobotController::configure(const yarp::os::Searchable& config,
 
     if (m_axesJointsCoupled)
     {
-        m_A.resize(m_numActuatedJoints, m_numActuatedAxis);
-        m_Bias.resize(m_numActuatedJoints, 1);
+        m_A = Eigen::MatrixXd::Identity(m_numActuatedAxis, m_numActuatedAxis);
+        m_Bias = Eigen::MatrixXd::Zero(m_numActuatedJoints, 1);
 
-        if (!m_doCalibration)
+        if (false && !m_doCalibration)
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/RobotController.cpp:68]: Probably this ``false`` can be removed. @RiccardoGrieco @lrapetti, do you remember why it was needed?

File: modules/HapticGlove_module/src/RobotSkin.cpp [hunk 6]
```diff
@@ -142,6 +130,100 @@ bool RobotSkin::configure(const yarp::os::Searchable& config,
         return false;
     }
 
+    const bool connectToCalibratedSkin
+        = config.check("connect_to_calibrated_skin", yarp::os::Value(0)).asBool();
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/RobotSkin.cpp:134]: Do you think this could work?
```suggestion
        = config.check("connect_to_calibrated_skin", yarp::os::Value(false)).asBool();
```

Comment (GiulioRomualdi) [reply modules/HapticGlove_module/src/RobotSkin.cpp:134]: yes it should work

**Comment by GiulioRomualdi** [comment]
The CI is failing but @S-Dafarra already fixed it in https://github.com/robotology/walking-teleoperation/pull/118. I would merge the PR even if the CI fails and then rebase [frugally_rebased](https://github.com/robotology/walking-teleoperation/tree/frugally_rebased) on top of master

**Comment by S-Dafarra** [comment]
If you prefer, we can merge this as is and apply the comments above in ``frugally_rebased``

**Comment by GiulioRomualdi** [comment]
This would simplify the merging procedure :)
